config/schema/myparcel/order/dhlforyou/nl_package_small.json

```diff
@@ -0,0 +1,25 @@
+{
+  "description": "order/dhlforyou/nl_package_small",
+  "properties": {
+    "deliveryOptions": {
+      "properties": {
+        "deliveryType": {
+          "enum": ["standard", "pickup", null]
+        },
+        "shipmentOptions": {
+          "properties": {
+            "ageCheck": {
+              "enum": [0, 1]
```

Comment:
Did you double check whether this enables these shipment options for pickup? Because I think this logic is already handled in `config/schema/myparcel/order/dhlforyou/base.json`? Ie why do we need this separately? (sanity check)